src/bivariate_normals.jl

```diff
@@ -18,9 +20,9 @@ immutable BvnComponent{NumType <: Number}
     z::NumType
     dsiginv_dsig::Matrix{NumType}
 
-    function BvnComponent{T1 <: Number, T2 <: Number, T3 <: Number}(
+    BvnComponent{T1 <: Number, T2 <: Number, T3 <: Number}(
         the_mean::Vector{T1}, the_cov::Matrix{T2}, weight::T3;
-        calculate_siginv_deriv::Bool=true)
+        calculate_siginv_deriv::Bool=true) = begin
```

Comment:
Agreed.  Sorry about this, I just missed this in the merge.

On Tue, Mar 8, 2016 at 10:39 AM, Kyle Barbary <notifications@github.com>
wrote:

> In src/bivariate_normals.jl
> <https://github.com/jeff-regier/Celeste.jl/pull/137#discussion_r55405316>:
>
> >          the_mean::Vector{T1}, the_cov::Matrix{T2}, weight::T3;
> > -        calculate_siginv_deriv::Bool=true)
> > +        calculate_siginv_deriv::Bool=true) = begin
>
> Let's use function syntax everywhere for functions rather than begin/end
> blocks.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/jeff-regier/Celeste.jl/pull/137/files#r55405316>.
>